tests/app/celery/test_tasks.py

```diff
@@ -1355,23 +1191,128 @@ def test_save_sms_uses_non_default_sms_sender_reply_to_text_if_provided(self, mo
         new_sender = service_sms_sender_dao.dao_add_sms_sender_for_service(service.id, "new-sender", False)
 
         notification = _notification_json(template, to="6502532222")
+        notification["sender_id"] = str(new_sender.id)
         mocker.patch("app.celery.provider_tasks.deliver_sms.apply_async")
 
         notification_id = uuid.uuid4()
-        save_sms(
-            service.id,
-            notification_id,
-            signer.sign(notification),
-            sender_id=new_sender.id,
-        )
-
+        save_smss(service.id, [signer.sign(notification)], notification_id)
         persisted_notification = Notification.query.one()
         assert persisted_notification.reply_to_text == "new-sender"
 
 
-class TestSaveEmail:
+class TestHandler:
```

Comment:
Can we please rename to TestSaveErrorHandling or something similar? it might not be clear which handler we are referring too otherwise.